rbi/annotations/mocha.rbi

```diff
@@ -16,11 +16,12 @@ end
 class Mocha::Expectation
   sig do
     params(
-      expected_parameters: T.untyped,
+      expected_parameters_or_matchers: T.untyped,
+      _arg1: T.untyped,
```

Comment:
👍  I've made the change, but I think because I deleted the branch after the PR was closed Github won't detect the change and wants me to open a new PR instead.